flexget/utils/cached_input.py

```diff
@@ -104,7 +104,7 @@ def db_cleanup(manager, session: DBSession) -> None:
         logger.verbose('Removed {} old input caches.', result)
 
 
-class cached:
+class Cached:
```

Comment:
Yeah, this seems like another one where usage is as if it's a function even though it's declared as a class.

Reply:
Yeah, noqa is good here.